test/models/InstanceTest.scala

```diff
@@ -0,0 +1,40 @@
+package models
+
+import org.scalatest.{FreeSpec, Matchers}
+import util.Fixtures._
+
+class InstanceTest extends FreeSpec with Matchers {
```

Comment:
`Instance` of what? These are testing `Stack`s at the moment, I think `StackTest` would be a batter name. That said, as I mentioned above, tests on model instances is a sign that data definitions and logic have been muddled together.